Nanyte25/RosterManager
# Node modules

# Issue link(s)

https://github.com/Nanyte25/RosterManager/issues/7

# What 

Remove node_modules that has been committed to source control

# Why

Remove bloat from repo

# How

Update .gitignore & remove previously watched node_module files from git cache 

# Verification Steps

Pull down changes, remove `node_modules` folder from directory, add again with `npm meteor install`, and confirm no node_modules files are picked up by `git status`

## Checklist:
